tests/test_sticker.py

```diff
@@ -367,14 +406,38 @@ def test_de_json(self, bot, sticker):
         assert sticker_set.name == name
         assert sticker_set.title == self.title
         assert sticker_set.is_animated == self.is_animated
+        assert sticker_set.is_video == self.is_video
         assert sticker_set.contains_masks == self.contains_masks
         assert sticker_set.stickers == self.stickers
         assert sticker_set.thumb == sticker.thumb
 
+    def test_create_sticker_set(self, bot, chat_id, video_sticker_file):
+        """Creates the sticker set (if needed) which is required for tests"""
+        test_by = f"test_by_{bot.username}"
+        for sticker_set in [test_by, f'animated_{test_by}', f'video_{test_by}']:
+            try:
+                bot.get_sticker_set(sticker_set)
+            except BadRequest as e:
+                if not e.message == "Stickerset_invalid":
+                    raise e
+
+                # We already have static and animated sticker sets, this is just for creating
+                # a video sticker set
```

Comment:
If this is only for the video stickers, why loop over all 3 cases above? I'd suggest to either keep it for just video or extend to all three - don't have a preference